modules/service/src/main/resources/db/migration/V0881__program_calibration_role.sql

```diff
@@ -0,0 +1,247 @@
+
+-- Programs may have a calibration role
+alter table t_program
+add column c_calibration_role e_calibration_role null;
+
+ALTER TYPE e_program_type ADD VALUE 'system';
+COMMIT;
+INSERT INTO t_program_type VALUES ('system', 'SYS', 'System', false);
+COMMIT;
```

Comment:
I needed this to use SYS lower on the same migration. Maybe I should split the migrations?

Reply:
That's what I've ended up doing (splitting the migrations).  It may be fine, I'm just not sure how this is handled with Flyway. Seems like it would be an error, but I don't know.  You could test it with the populate from dev script though.

Reply:
This was solved, the migrations was split and I removed the `COMMIT` statements